library/waku_thread/inter_thread_communication/requests/peer_manager_request.nim

```diff
@@ -1,29 +1,49 @@
 import std/[sequtils, strutils]
 import chronicles, chronos, results
-import ../../../../waku/factory/waku, ../../../../waku/node/waku_node, ../../../alloc
+import
+  ../../../../waku/factory/waku,
+  ../../../../waku/node/waku_node,
+  ../../../alloc,
+  ../../../../waku/node/peer_manager
 
-type PeerManagementMsgType* = enum
+type PeerManagementMsgType* {.pure.} = enum
   CONNECT_TO
+  GET_PEER_IDS_FROM_PEER_STORE
```

Comment:
Maybe `GET_ALL_PEER_IDS` ?

Reply:
Much better thanks! Added in commit: https://github.com/waku-org/nwaku/pull/3026/commits/9268abbb1ddc006dc909bc8c04ede3b7054a61fd